src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -82,7 +82,7 @@ export class TSConfigReader extends OptionsComponent
             delete compilerOptions[key];
         }
 
-        _.merge(event.data, compilerOptions);
+        _.defaults(event.data, compilerOptions);
 
         if ("typedocOptions" in data) {
             _.merge(event.data, data.typedocOptions);
```

Comment:
What do you think about this one too? Seems like this would also be better with `_.defaults` for the same reason. In which case, this whole thing can probably change to:

```ts
_.defaults(event.data, data.raw.typedocOptions, compilerOptions)
```